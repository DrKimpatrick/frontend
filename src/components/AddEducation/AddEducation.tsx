import React, { FC } from 'react';
import { withRouter } from 'react-router-dom';
import './AddEducation.scss';
import ArrowBackTwoToneIcon from '@material-ui/icons/ArrowBackTwoTone';
import ArrowRightAltTwoToneIcon from '@material-ui/icons/ArrowRightAltTwoTone';
import DatePicker from 'components/DatePicker/DatePicker';

type props = {};
const AddEducation: FC<props> = () => {
  const options = [
    { value: 'Software Engineer', label: 'Software Engineer' },
    { value: 'Product Manager', label: 'Product Manager' },
    { value: 'Product Designer', label: 'Product Designer' },
    { value: 'Software Engineer1', label: 'Software Engineer1' },
    { value: 'Product Manager1', label: 'Product Manager1' },
    { value: 'Product Designer1', label: 'Product Designer1' },
    { value: 'Software Engineer2', label: 'Software Engineer2' },
    { value: 'Product Manager2', label: 'Product Manager2' },
    { value: 'Product Designer2', label: 'Product Designer2' }
  ];

  return (
    <section className="add-education-section w-1/3 m-auto text-textGray">
      <div className="flex relative h-auto my-8">
        <div className="back-arrow cursor-pointer">
          <ArrowBackTwoToneIcon />
        </div>
        <h1 className="font-bold text-xl title">
          What was the last school you attended?
        </h1>
      </div>
      <div className="text-textGray mt-8">
        <input
          type="text"
          className="border outline-none bg-transparent rounded-sm w-full px-3 text-textGray input-height"
          placeholder="School Name"
        />
      </div>

      <div className="text-textGray mt-4">
        <input
          type="text"
          className="border outline-none bg-transparent rounded-sm w-full px-3 text-textGray input-height"
          placeholder="Level"
        />
      </div>
      <div className="text-textGray mt-4">
        <input
          type="text"
          className="border outline-none bg-transparent rounded w-full px-3 text-textGray input-height"
          placeholder="Specializations"
        />
      </div>
      <div className="text-textGray mt-4">
        <input
          type="text"
          className="border outline-none bg-transparent rounded w-full px-3 text-textGray input-height"
          placeholder="Degree or Certification Received"
        />
      </div>

      <div className="flex justify-between text-textGray mt-4">
        <DatePicker label="Start Date" defaultValue="2017-05-24" />
        <DatePicker label="Send Date" defaultValue="22017-05-24" />
      </div>

      <div className="text-textGray mt-4">
        <textarea
          name="accomplishments"
          placeholder="Accomplishments"
          className="border outline-none bg-transparent rounded w-full p-3 text-textGray h-48"
          cols={30}
          rows={7}
        />
      </div>

      <div className="flex justify-center mt-12">
        <button
          data-testid="next-button"
          className="next-btn text-white hover:bg-gray-800 font-semibold py-1 px-3 w-32 rounded-sm shadow flex justify-around"
        >
          <span className="">Next</span> <ArrowRightAltTwoToneIcon />
        </button>
      </div>
    </section>
  );
};

export default withRouter(AddEducation);
