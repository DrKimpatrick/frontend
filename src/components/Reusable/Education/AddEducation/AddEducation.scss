.add-education-section {
  .title {
    font-size: 22px;
    color: #747474;
    text-decoration: none solid rgb(116, 116, 116);
  }

  .back-arrow {
    position: absolute;
    color: #747474;
    left: -43px;
    top: 0px;
    font-size: 22px;
  }

  .input-height {
    height: 47px;
    border-radius: 2px;
    color: #747474;

    &::placeholder {
      color: #c9c9c9;
    }
  }
}

@media (max-width: 786px) {
  .add-education-section {
    width: 50%;
  }
}

@media (max-width: 500px) {
  .add-education-section {
    width: 90%;

    .title {
      text-align: center;
    }
  }
}
