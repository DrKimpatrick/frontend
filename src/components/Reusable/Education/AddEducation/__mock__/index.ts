export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  messages: {
    message: 'sent'
  },
  educations: {
    loading: false,
    errors: {
      errors: null
    },
    education: {
      _id: 'id',
      schoolName: 'school name',
      level: 'level',
      degreeOrCertification: 'education',
      specializations: ['specialization'],
      startDate: '2020-10-10',
      endDate: '2020-10-5',
      accomplishments: ['snow'],
      isCurrentEducation: false,
      verificationStatus: 'verified'
    }
  }
};
