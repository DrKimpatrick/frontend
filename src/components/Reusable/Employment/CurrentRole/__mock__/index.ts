export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  employments: {
    employment: {
      companyName: 'snow'
    },
    errors: null,
    loading: false
  },
  messages: {
    message: 'sent'
  },
  educations: {
    education: {
      schoolName: 'schoolName'
    }
  }
};
