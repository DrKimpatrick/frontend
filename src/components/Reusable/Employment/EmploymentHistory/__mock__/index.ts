export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  employments: {
    employment: {
      companyName: 'snow',
      supervisor: 'supervisor',
      title: 'title',
      startDate: '2020-10-5',
      responsibilities: ['res'],
      accomplishments: ['res'],
      favoriteProject: 'res',
      skillsUsed: []
    },
    errors: null,
    loading: false
  },
  messages: {
    message: 'sent'
  },
  educations: {
    education: {
      schoolName: 'schoolName'
    }
  },
  skills: {
    userSkill: [
      {
        supervisor: {
          detail: {
            name: 'user',
            email: 'email@gmail.com',
            phoneNumber: '0786601003'
          },
          name: 'HR'
        },
        skillsUsed: [],
        responsibilities: [],
        accomplishments: [],
        verificationStatus: 'unverified',
        isCurrentPosition: true,
        _id: '5fcf65a2c4c797a20acc1c99',
        title: 'Product Manager',
        startDate: '2020-10-15T00:00:00.000Z',
        endDate: null,
        companyName: 'heza',
        favoriteProject: 'tembea',
        userId: '5fcdf47ea1d1dd1564a21613',
        createdAt: '2020-12-08T11:38:10.729Z',
        updatedAt: '2020-12-08T11:38:10.729Z'
      }
    ]
  }
};
