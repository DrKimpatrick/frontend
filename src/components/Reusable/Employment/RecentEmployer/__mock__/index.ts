export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  employments: {
    employment: {
      companyName: 'snow',
      supervisor: 'supervisor',
      title: 'title',
      startDate: '2020-10-5',
      responsibilities: ['res'],
      accomplishments: ['res'],
      favoriteProject: 'res',
      _id: 'id'
    },
    errors: null,
    loading: false,
    submitLoading: false
  },
  messages: {
    message: 'sent'
  },
  educations: {
    education: {
      schoolName: 'schoolName'
    }
  },
  skills: {
    userSkill: [
      {
        skill: {
          _id: '5fcf1c5b11c16c41f7c6c723',
          skill: 'Python',
          createdAt: '2020-12-08T06:25:31.128Z',
          updatedAt: '2020-12-08T06:25:31.128Z'
        },
        level: 'intermediate',
        verificationStatus: 'verified',
        _id: '5fcf52105cd56b9843252696',
        createdAt: '2020-12-08T10:14:40.231Z',
        updatedAt: '2020-12-08T13:05:46.873Z'
      }
    ]
  }
};
