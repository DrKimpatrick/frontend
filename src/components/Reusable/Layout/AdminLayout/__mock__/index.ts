export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {}
  }
};
