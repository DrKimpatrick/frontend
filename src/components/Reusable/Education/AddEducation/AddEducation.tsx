import React, { FC, useEffect } from 'react';
import { useSelector, useDispatch } from 'react-redux';
import { withRouter, useHistory } from 'react-router-dom';
import { ArrowBackTwoTone } from '@material-ui/icons';
import { EducationInput, InitialEducationValue } from 'components/Reusable';
import { addEducation } from 'redux/actions/education';
import { format } from 'date-fns';
import { RootState } from 'redux/store';
import NavBar from 'components/Reusable/Layout/NavBar/NavBar';
import { MainBackground } from 'components/Reusable/Layout/MainBackground';
import './AddEducation.scss';

const AddEducation: FC = () => {
  const dispatch = useDispatch();

  const history = useHistory();

  const reducer = useSelector((state: RootState) => {
    const { loading, errors, education } = state.educations;

    const { message } = state.messages;

    return { message, loading, errors, education };
  });

  useEffect(() => {
    if (reducer.message && reducer.education) {
      history.push({
        pathname: '/education-history',
        state: { educationId: reducer.education._id }
      });
    }
  }, [reducer.message, reducer.education, history]);

  return (
    <>
      <NavBar />
      <div className="containers">
        <div className="recent-employer-section w-1/3 m-auto text-textGray">
          <div className="flex relative h-auto my-8">
            <button className="back-arrow cursor-pointer" type="button">
              <ArrowBackTwoTone />
            </button>
            <h1 className="font-bold text-base title">
              What was the last school you attended?
            </h1>
          </div>
          <EducationInput
            initialValue={{
              schoolName: '',
              level: '',
              degreeOrCertification: '',
              isCurrentEducation: false,
              specializations: [],
              specialization: '',
              accomplishments: [],
              accomplishment: '',
              startDate: '',
              endDate: ''
            }}
            submit={(values: InitialEducationValue) => {
              const { startDate, endDate } = values;
              addEducation({
                ...values,
                startDate: format(new Date(startDate), 'yyyy-MM-dd'),
                endDate: format(new Date(String(endDate)), 'yyyy-MM-dd')
              })(dispatch);
            }}
            backendErrors={
              reducer.errors && reducer.errors.errors && reducer.errors.errors
            }
            loading={!!reducer.loading}
            buttonName="Next"
          />
        </div>
      </div>
      <MainBackground />
    </>
  );
};

export default withRouter(AddEducation);
