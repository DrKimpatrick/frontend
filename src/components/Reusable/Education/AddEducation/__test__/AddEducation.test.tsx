import { cleanup, fireEvent, render, waitFor } from '@testing-library/react';
import React from 'react';
import ReactDom from 'react-dom';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import thunk from 'redux-thunk';
import configureMockStore from 'redux-mock-store';
import AddEducation from '../AddEducation';
import renderer from 'react-test-renderer';
import { initialState } from '../__mock__';

const mockHistoryPush = jest.fn();

let store: any;

const middleware = [thunk];

const mockStore = configureMockStore(middleware);

jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useHistory: () => ({
    push: mockHistoryPush
  })
}));

describe('`AddEducation` component', () => {
  beforeEach(() => {
    store = mockStore(initialState);
  });

  afterEach(cleanup);
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDom.render(
      <Provider store={store}>
        <Router>
          <AddEducation />
        </Router>
      </Provider>,
      div
    );
  });

  it('matches the snapshot', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Router>
            <AddEducation />
          </Router>
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should allow user to add education', async () => {
    const { container } = render(
      <Provider store={store}>
        <Router>
          <AddEducation />
        </Router>
      </Provider>
    );

    const schoolName: Element | any = container.querySelector(
      'input[name="schoolName"]'
    );

    const level: Element | any = container.querySelector('input[name="level"]');

    const degree: Element | any = container.querySelector(
      'input[name="degreeOrCertification"]'
    );

    const startDate: Element | any = container.querySelector(
      'input[name="startDate"]'
    );

    const endDate: Element | any = container.querySelector(
      'input[name="endDate"]'
    );

    const currentEducation: Element | any = container.querySelector(
      'input[name="isCurrentEducation"]'
    );

    const form: Element | any = container.querySelector('form');

    await waitFor(() => {
      fireEvent.change(schoolName, {
        target: {
          value: 'schoolname'
        }
      });
    });

    await waitFor(() => {
      fireEvent.change(level, {
        target: {
          value: 'level'
        }
      });
    });

    await waitFor(() => {
      fireEvent.change(degree, {
        target: {
          value: 'degree'
        }
      });
    });

    await waitFor(() => {
      fireEvent.change(startDate, {
        target: {
          value: '2020-10-05'
        }
      });
    });

    await waitFor(() => {
      fireEvent.change(endDate, {
        target: {
          value: '2020-10-05'
        }
      });
    });

    await waitFor(() => {
      fireEvent.change(currentEducation, {
        target: {
          value: true
        }
      });
    });

    await waitFor(() => {
      fireEvent.submit(form);
    });

    expect(mockHistoryPush).toHaveBeenCalledWith({
      pathname: '/education-history',
      state: { educationId: 'id' }
    });
  });
});
