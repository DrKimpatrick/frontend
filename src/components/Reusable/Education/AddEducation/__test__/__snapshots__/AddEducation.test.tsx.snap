// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`AddEducation\` component matches the snapshot 1`] = `
Array [
  <nav
    className="flex items-center bg-white-800 px-20 py-5 flex-wrap"
  >
    <a
      className="text-gray-800"
      href="/"
      onClick={[Function]}
    >
      <img
        alt="logo"
        className="logo w-24"
        src="logo-image.png"
      />
    </a>
    <div
      className="hidden top-navbar w-full lg:inline-flex lg:flex-grow lg:w-auto"
      id="navigation"
    >
      <div
        className="lg:inline-flex lg:flex-row lg:ml-auto lg:w-auto w-full lg:items-center items-start flex flex-col lg:h-auto"
      >
        <div>
          <a
            className="lg:inline-flex lg:w-auto w-full px-8 py-2 rounded text-gray-600 items-center justify-center hover:text-gray-700"
            href="/"
            onClick={[Function]}
          >
            <span>
              How it works
            </span>
          </a>
          <a
            className="lg:inline-flex lg:w-auto w-full px-8 py-2 rounded text-gray-600 items-center justify-center hover:text-gray-700"
            href="/"
            onClick={[Function]}
          >
            <span>
              About us
            </span>
          </a>
        </div>
        <div
          className="lg:inline-flex items-center undefined"
        >
          <div
            className="px-8 lg:inline-flex items-center"
          >
            <div
              className="makeStyles-root-1"
            >
              <div>
                <div
                  aria-haspopup="true"
                  className="notification-container"
                  onClick={[Function]}
                  onTouchEnd={[Function]}
                >
                  <div
                    className="text-gray-600 relative notification-icon"
                    onClick={[Function]}
                  >
                    <div
                      className="bg-white text-red-600 h-3 w-3 rounded-full absolute right-0 top-0 flex items-center justify-center border-red-600 border border-solid"
                      style={
                        Object {
                          "fontSize": "6px",
                        }
                      }
                    >
                      <span>
                        9+
                      </span>
                    </div>
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root cursor-pointer"
                      focusable="false"
                      style={
                        Object {
                          "color": "#ff0000",
                        }
                      }
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 22c1.1 0 2-.9 2-2h-4c0 1.1.89 2 2 2zm6-6v-5c0-3.07-1.64-5.64-4.5-6.32V4c0-.83-.67-1.5-1.5-1.5s-1.5.67-1.5 1.5v.68C7.63 5.36 6 7.92 6 11v5l-2 2v1h16v-1l-2-2z"
                      />
                    </svg>
                  </div>
                  <div
                    className="header-popup tri-right border right-top hide"
                  >
                    <div
                      className="text-gray-600 w-56 space-x-2 text-sm py-2 flex "
                    >
                      <span
                        className="text-red-600 font-bold"
                      >
                        1
                      </span>
                      <div
                        className="flex flex-col w-full flex-wrap"
                      >
                        <span
                          className="text-red-600 font-bold"
                        >
                          You have been verified
                        </span>
                        <p
                          className="break-normal"
                        >
                          Check your email for the link verification
                        </p>
                      </div>
                    </div>
                    <div
                      className="text-gray-600 w-56 space-x-2 text-sm py-2 flex "
                    >
                      <span
                        className="text-red-600 font-bold"
                      >
                        1
                      </span>
                      <div
                        className="flex flex-col w-full flex-wrap"
                      >
                        <span
                          className="text-red-600 font-bold"
                        >
                          You have been verified
                        </span>
                        <p
                          className="break-normal"
                        >
                          Check your email for the link verification
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              className="text-gray-600 relative"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M11.99 18.54l-7.37-5.73L3 14.07l9 7 9-7-1.63-1.27-7.38 5.74zM12 16l7.36-5.73L21 9l-9-7-9 7 1.63 1.27L12 16z"
                />
              </svg>
            </div>
          </div>
          <div
            className="text-gray-600"
          >
            <span>
              username
            </span>
          </div>
          <div
            className="makeStyles-root-1"
          >
            <div>
              <div
                aria-haspopup="true"
                className="profile-container"
                onClick={[Function]}
                onTouchEnd={[Function]}
              >
                <div
                  aria-haspopup="true"
                  onClick={[Function]}
                >
                  <img
                    alt="avatar"
                    className="inline-block ml-2 h-12 w-12 rounded-full text-white shadow-solid cursor-pointer"
                    src="https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png"
                  />
                </div>
                <div
                  className="header-popup tri-left border right-top hide"
                >
                  <ul
                    className="MuiList-root MuiList-padding"
                    onKeyDown={[Function]}
                    role="menu"
                    tabIndex={-1}
                  >
                    <li
                      aria-disabled={false}
                      className="MuiButtonBase-root MuiListItem-root MuiMenuItem-root text-gray-600 MuiMenuItem-gutters MuiListItem-gutters MuiListItem-button"
                      onBlur={[Function]}
                      onClick={[Function]}
                      onDragLeave={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onKeyUp={[Function]}
                      onMouseDown={[Function]}
                      onMouseLeave={[Function]}
                      onMouseUp={[Function]}
                      onTouchEnd={[Function]}
                      onTouchMove={[Function]}
                      onTouchStart={[Function]}
                      role="menuitem"
                      tabIndex={0}
                    >
                      <div
                        className="text-gray-600 w-56 space-x-3 text-sm py-2"
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M15.95 10.78c.03-.25.05-.51.05-.78s-.02-.53-.06-.78l1.69-1.32c.15-.12.19-.34.1-.51l-1.6-2.77c-.1-.18-.31-.24-.49-.18l-1.99.8c-.42-.32-.86-.58-1.35-.78L12 2.34c-.03-.2-.2-.34-.4-.34H8.4c-.2 0-.36.14-.39.34l-.3 2.12c-.49.2-.94.47-1.35.78l-1.99-.8c-.18-.07-.39 0-.49.18l-1.6 2.77c-.1.18-.06.39.1.51l1.69 1.32c-.04.25-.07.52-.07.78s.02.53.06.78L2.37 12.1c-.15.12-.19.34-.1.51l1.6 2.77c.1.18.31.24.49.18l1.99-.8c.42.32.86.58 1.35.78l.3 2.12c.04.2.2.34.4.34h3.2c.2 0 .37-.14.39-.34l.3-2.12c.49-.2.94-.47 1.35-.78l1.99.8c.18.07.39 0 .49-.18l1.6-2.77c.1-.18.06-.39-.1-.51l-1.67-1.32zM10 13c-1.65 0-3-1.35-3-3s1.35-3 3-3 3 1.35 3 3-1.35 3-3 3z"
                            transform="scale(1.2, 1.2)"
                          />
                        </svg>
                        <span>
                          Settings
                        </span>
                      </div>
                      <span
                        className="MuiTouchRipple-root"
                      />
                    </li>
                    <li
                      aria-disabled={false}
                      className="MuiButtonBase-root MuiListItem-root MuiMenuItem-root text-gray-600 MuiMenuItem-gutters MuiListItem-gutters MuiListItem-button"
                      onBlur={[Function]}
                      onClick={[Function]}
                      onDragLeave={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onKeyUp={[Function]}
                      onMouseDown={[Function]}
                      onMouseLeave={[Function]}
                      onMouseUp={[Function]}
                      onTouchEnd={[Function]}
                      onTouchMove={[Function]}
                      onTouchStart={[Function]}
                      role="menuitem"
                      tabIndex={-1}
                    >
                      <div
                        className="text-gray-600 w-56 space-x-3 text-sm py-2"
                        onClick={[Function]}
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M13 3h-2v10h2V3zm4.83 2.17l-1.42 1.42C17.99 7.86 19 9.81 19 12c0 3.87-3.13 7-7 7s-7-3.13-7-7c0-2.19 1.01-4.14 2.58-5.42L6.17 5.17C4.23 6.82 3 9.26 3 12c0 4.97 4.03 9 9 9s9-4.03 9-9c0-2.74-1.23-5.18-3.17-6.83z"
                          />
                        </svg>
                        <span>
                          Logout
                        </span>
                      </div>
                      <span
                        className="MuiTouchRipple-root"
                      />
                    </li>
                  </ul>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </nav>,
  <div
    className="containers"
  >
    <div
      className="recent-employer-section w-1/3 m-auto text-textGray"
    >
      <div
        className="flex relative h-auto my-8"
      >
        <button
          className="back-arrow cursor-pointer"
          type="button"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"
            />
          </svg>
        </button>
        <h1
          className="font-bold text-base title"
        >
          What was the last school you attended?
        </h1>
      </div>
      <form
        autoComplete="off"
        onSubmit={[Function]}
      >
        <div
          className="text-textGray mt-8"
        >
          <input
            className="border outline-none bg-transparent rounded-sm w-full px-3 text-textGray input-height"
            name="schoolName"
            onChange={[Function]}
            placeholder="School Name"
            type="text"
            value=""
          />
        </div>
        <div
          className="text-textGray mt-4"
        >
          <input
            className="border outline-none bg-transparent rounded w-full px-3 text-textGray input-height"
            name="level"
            onChange={[Function]}
            placeholder="Level"
            type="text"
            value=""
          />
        </div>
        <div
          className="text-textGray mt-4"
        >
          <input
            className="border outline-none bg-transparent rounded w-full px-3 text-textGray input-height"
            name="degreeOrCertification"
            onChange={[Function]}
            placeholder="Degree or Certification Received"
            type="text"
            value=""
          />
        </div>
        <div
          className="flex justify-between text-textGray mt-4 dateContainer"
        >
          <div
            className="item"
          >
            <label
              htmlFor="start date"
            >
              Start date
            </label>
            <input
              id="date"
              name="startDate"
              onChange={[Function]}
              type="date"
              value=""
            />
          </div>
          <div
            className="item"
          >
            <label
              htmlFor="end date"
            >
              End date
            </label>
            <input
              id="date"
              name="endDate"
              onChange={[Function]}
              type="date"
              value=""
            />
          </div>
        </div>
        <div
          className="text-textGray flex"
          style={
            Object {
              "alignItems": "center",
            }
          }
        >
          <div
            style={
              Object {
                "flexGrow": 1,
                "width": "50%",
              }
            }
          />
          <div
            style={
              Object {
                "flexGrow": 1,
                "width": "50%",
              }
            }
          />
        </div>
        <div
          className="text-textGray flex"
          style={
            Object {
              "alignItems": "center",
            }
          }
        >
          <span
            aria-disabled={false}
            className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-2 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={null}
          >
            <span
              className="MuiIconButton-label"
            >
              <input
                aria-label="uncontrolled-checkbox"
                checked={false}
                className="PrivateSwitchBase-input-5"
                data-indeterminate={false}
                name="isCurrentEducation"
                onChange={[Function]}
                type="checkbox"
              />
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                />
              </svg>
            </span>
            <span
              className="MuiTouchRipple-root"
            />
          </span>
          <label
            htmlFor=""
          >
            I am currently studying here
          </label>
        </div>
        <div
          className="text-textGray mt-4 divider"
          style={
            Object {
              "border": "none",
            }
          }
        >
          <div
            className="responsibilities addBorder"
          >
            <input
              className="outline-none bg-transparent rounded w-full px-3 text-textGray input-height"
              name="specialization"
              onChange={[Function]}
              placeholder="Specializations"
              type="text"
              value=""
            />
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-text add"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          className="text-textGray mt-4 divider"
          style={
            Object {
              "border": "none",
            }
          }
        >
          <div
            className="responsibilities addBorder"
          >
            <input
              className="outline-none bg-transparent rounded w-full px-3 text-textGray input-height"
              name="accomplishment"
              onChange={[Function]}
              placeholder="Accomplishments"
              type="text"
              value=""
            />
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-text add"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          className="flex justify-center mt-12"
        >
          <button
            className="next-btn text-white hover:bg-gray-800 font-semibold py-1 px-3 w-32 rounded-sm shadow flex justify-around"
            data-testid="next-button"
            disabled={false}
            type="submit"
          >
            <span
              className=""
            >
              Next
            </span>
             
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M16.01 11H4v2h12.01v3L20 12l-3.99-4v3z"
              />
            </svg>
          </button>
        </div>
      </form>
    </div>
  </div>,
  <div
    className="mainBackground"
  >
    <img
      alt=""
      src="motif-image.png"
      style={
        Object {
          "backgroundColor": "none",
        }
      }
    />
  </div>,
]
`;
